firewood/src/merkle.rs

```diff
@@ -1308,136 +1299,21 @@ pub fn from_nibbles(nibbles: &[u8]) -> impl Iterator<Item = u8> + '_ {
 }
 
 #[cfg(test)]
-mod test {
+mod tests {
     use super::*;
-    use shale::cached::{DynamicMem, PlainMem};
-    use shale::{CachedStore, Storable};
-    use std::ops::Deref;
+    use node::tests::{extension, leaf};
+    use shale::{cached::DynamicMem, compact::CompactSpace, CachedStore};
     use std::sync::Arc;
     use test_case::test_case;
 
     #[test_case(vec![0x12, 0x34, 0x56], vec![0x1, 0x2, 0x3, 0x4, 0x5, 0x6])]
     #[test_case(vec![0xc0, 0xff], vec![0xc, 0x0, 0xf, 0xf])]
-    fn test_to_nibbles(bytes: Vec<u8>, nibbles: Vec<u8>) {
+    fn to_nibbles(bytes: Vec<u8>, nibbles: Vec<u8>) {
         let n: Vec<_> = bytes.into_iter().flat_map(to_nibble_array).collect();
         assert_eq!(n, nibbles);
     }
 
-    const ZERO_HASH: TrieHash = TrieHash([0u8; TRIE_HASH_LEN]);
-
-    #[test]
-    fn test_hash_len() {
-        assert_eq!(TRIE_HASH_LEN, ZERO_HASH.dehydrated_len() as usize);
```

Comment:
This test was deleted. It's impossible to create a `TrieHash` who's length is different than `TRIE_HASH_LENGTH` now. 

I don't think tests exist for checking that the actual `dehydrated_len` is the serialized length (assuming that's what this means), but this test didn't test that either.